src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -106,13 +106,16 @@ const SidebarSidecarLayoutContent = ({
                 />
               )}
             </main>
+            <div className={s.sidecar}>
+              <SidecarContent />
+            </div>
+          </div>
+          <div className={s.footerArea}>
             <Footer
               className={s.footer}
               openConsentManager={openConsentManager}
             />
-          </div>
-          <div className={`${s.sidecar} g-hide-on-mobile g-hide-on-tablet`}>
-            <SidecarContent />
+            <div className={s.footerSpaceForSidecar}></div>
```

Comment:
**Nitpick ⛏️:**  I think this fill could be done via CSS vs needing the extra empty element in the dom.

<img width="1221" alt="CleanShot 2022-05-13 at 12 30 31@2x" src="https://user-images.githubusercontent.com/825855/168327496-554f8173-ab96-4aab-b346-60bd28c5b753.png">

Applying the same styles to the `footerArea` after element and adding gap since were in a flex container.

```css
.footerArea {
  gap: var(--sidecar-margin-left);
  
  &::after {
    content: '';
    display: block;
    width: var(--sidecar-width);
    background: red;
  }
}
```

Reply:
ooh nice! ✅  updated, I dig this.

this made me notice we can use `gap` for spacing the main area with the actual sidecar as well, so updated that too 👍